src/BingoGame.Indexer.CA/Entities/BingoGameIndex.cs

```diff
@@ -0,0 +1,26 @@
+using AElf.Indexing.Elasticsearch;
+using AElfIndexer.Client;
+using Nest;
+using BingoGame.Indexer.CA.GraphQL;
+
+namespace BingoGame.Indexer.CA.Entities;
+
+public class BingoGameIndex : AElfIndexerClientEntity<string>, IIndexBuild
```

Comment:
The name is wrong. It's not an index. Let's call it `BingoGameIndexEntry`.

Reply:
noted, changed all BingoGameIndex in the project to BingoGameIndexEntry